source/module_hamilt_lcao/hamilt_lcaodft/FORCE_gamma.cpp

```diff
@@ -104,7 +105,7 @@ void Force_LCAO<double>::allocate(const UnitCell& ucell,
                               orb,
                               pv,
                               two_center_bundle,
-                              &GlobalC::GridD,
+                              &gd,
```

Comment:
for most GlobalC::GridD, could we add 'const' before we passing the variable?

Reply:
I will do it in later PRs.